database/seed.js

```diff
@@ -19,6 +19,24 @@ const sampleData = () => {
           fullName: faker.name.findName(),
           created_At: faker.date.past(),
           image: faker.image.avatar(),
```

Comment:
rename image to avatar

Reply:
Updated image to userImage for better name